pkg/cmd/server/api/types.go

```diff
@@ -450,6 +450,11 @@ type MasterNetworkConfig struct {
 	// CIDR will be rejected. Rejections will be applied first, then the IP checked against one of the allowed CIDRs. You
 	// should ensure this range does not overlap with your nodes, pods, or service CIDRs for security reasons.
 	ExternalIPNetworkCIDRs []string
+	// IngressIPNetworkCIDRs controls the range to assign ingress ips from for services of type LoadBalancer on bare
+	// metal. If empty, ingress ips will not be assigned. It may contain a single CIDR that will be allocated from.
+	// For security reasons, you should ensure that this range does not overlap with the CIDRs reserved for external ips,
+	// nodes, pods, or services.
+	IngressIPNetworkCIDRs []string
```

Comment:
I assumed it would be desirable to be able to specify the range more flexibly in the future (in the same way as ExternalIPNetworkCIDRs), and that it would be preferable to relax the restriction on a single CIDR rather than having to change the name and type of the parameter to do so.  Is future flexibility desirable or should I do as you suggest?